File: MajorityElement/majorityElement.cpp
class Solution {
public:
    int majorityElement(vector<int>& nums) {
        std::map<int,int> M;
        for (int i{0}; i<nums.size();i++)
        {
            if (M.find(nums[i]) == M.end()) M[nums[i]] =1;
            else M[nums[i]] += 1;
        }
        for(auto& it:M)
        {
            if(it.second >(nums.size()/2))
                return it.first;
        }
        return 1;
    }
};

File: MajorityElement/majorityelement.py
class Solution(object):
    def majorityElement(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        from collections import Counter
        standard_for_greater = len(nums)/2
        a = Counter(nums)
        key = [k for k,v in a.items() if v>standard_for_greater][0]
        return key
        